devicetypes/smartthings/aeon-multisensor.src/aeon-multisensor.groovy

```diff
@@ -137,12 +137,13 @@ def parse(String description)
 // Event Generation
 def zwaveEvent(physicalgraph.zwave.commands.wakeupv1.WakeUpNotification cmd)
 {
-	def results = [createEvent(descriptionText: "${device.displayName} woke up", isStateChange: false)]
+	def results = []
+	results << createEvent(descriptionText: "${device.displayName} woke up", isStateChange: false)
 	// If we haven't configured yet, then do so now
 	if (!isConfigured()) {
-		results += configure()
+		results << configure()
```

Comment:
Since `configure()` returns commands, it has to be in the `response()` call to be bundled together as a HubMultiAction with the wakeUpNMI. See how Aeon Multisensor 6 does it with `result` vs `cmds`